sentry/src/main/java/io/sentry/SentryBaseEvent.java

```diff
@@ -106,6 +107,21 @@ public void setRequest(final @Nullable Request request) {
     return throwable;
   }
 
+  /**
+   * Returns the captured Throwable or null. If a throwable is wrapped in {@link
+   * ExceptionMechanismException}, returns unwrapped throwable.
+   *
+   * @return the Throwable or null
+   */
+  public @Nullable Throwable getOriginThrowable() {
```

Comment:
we could remove the code dup from `DuplicateEventDetectionEventProcessor` using this method now

Reply:
True! Will be removed in few mins.